src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -615,20 +611,68 @@ def launch_fluent(
 
     del start_container
 
-    if start_watchdog is None and cleanup_on_exit:
+    if additional_arguments is None:
+        additional_arguments = ""
+    elif fluent_launch_mode == LaunchMode.PIM:
```

Comment:
In later changes, fluent_launch_mode could be a key `launcher_factory.create(key)` then each launcher could be responsible for constraining/validating its own launch configuration in `launcher.launch(<args>)`. E.g., `pim_launcher.launch(<args>)` would not have an `additional_arguments` argument.